apps/ae_backend_service/lib/backend_session.ex

```diff
@@ -20,8 +20,12 @@ defmodule BackendSession do
             port: nil,
             game: %{},
             responder_contract: nil,
+            fp_timer: nil,
             expected_state: nil
 
+  # TODO: this should be configurable, also actual mine rate is unpredictable...
+  @blocks_reaction_time 10
+  @mine_rate 10000
```

Comment:
default is about 3 minutes
```suggestion
  @mine_rate 180000
```